app_dart/lib/src/foundation/github_checks_util.dart

```diff
@@ -40,6 +40,27 @@ class GithubChecksUtil {
     );
   }
 
+  /// Finds all check suites that are associated with a given git [ref].
+  Future<Stream<github.CheckSuite>> listCheckSuitesForRef(
```

Comment:
Tests for this method?

Reply:
Can you suggest how I should test this? I added a test for `searchIssuesAndPRs` to validate that it encodes the query properly, but for `listCheckSuitesForRef` and `getPullRequest` I can't think of anything since they just forward the call unconditionally.